spec/openapi.yaml

```
@@ -218,36 +301,25 @@ components:
         - name
         - identifiers
         - versions
-    component-version:
+    #
+    # TEA Release and related objects
+    #
+    release:
       type: object
-      description: A specific version of a TEA component
       properties:
-        collection_uuid:
-          "$ref": "#/components/schemas/uuid"
+        uuid:
+          "$ref": "#/components/schemas/type_uuid"
         version:
           type: string
-          description: Version number of TEA component
-        identifiers:
-          type: array
-          description: List of identifiers for this component version
-          items:
-            "$ref": "#/components/schemas/identifier"
         release_date:
-          "$ref": "#/components/schemas/date-time"
-          description: Release date
-        pre_release:
-          type: boolean
-          description: Marks if the version is a pre-release version
-        classifier:
-          type: object
-          description: Optional classifier to distinguish between artifacts
+          type: string
+          format: date-time
+          example: '2024-03-20T15:30:00Z'
```

```suggestion
          "$ref": "#/components/schemas/date-time"
```

The idea is to use a common timestamp format that complies both to [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) and ISO 8601.